lms/envs/common.py

```diff
@@ -1611,8 +1595,6 @@ def _make_mako_template_dirs(settings):
 # This should be moved into an XBlock Runtime/Application object
 # once the responsibility of XBlock creation is moved out of modulestore - cpennington
 XBLOCK_MIXINS = (LmsBlockMixin, InheritanceMixin, XModuleMixin, EditInfoMixin)
-if SkillTaggingMixin:
-    XBLOCK_MIXINS += (SkillTaggingMixin,)
```

Comment:
@andy-armstrong: Were you saying that this is what needs to be reverted?

Reply:
In other words, should we leave the setting in place if that could cause more harm?